app.js

```diff
@@ -73,89 +72,96 @@ function _init() {
     && config.server.ssl_key && config.server.ssl_port;
 }
 
-function refineRemotes(myRemotes) {
-  var newRemotes = {};
-  var newRemoteCommands = null;
-  var remote = null;
-
-  function isBlacklistExisting(remoteName) {
-    return config.blacklists && config.blacklists[remoteName];
+function overrideConfigurationForDebugOrDevelopment() {
+  var lircTest;
+  if (process.env.npm_package_config_test_env) {
+    console.log('we are in test mode!');
```

Comment:
Does this line display each time the tests are run? If so, might want to remove this so it's not as noisy.